tests/Organizer/ReadModel/RDF/OrganizerJsonToTurtleConverterTest.php

```diff
@@ -73,6 +76,7 @@ protected function setUp(): void
             new OrganizerDenormalizer(),
             $addressParser,
             $this->imageNormalizer,
+            new RdfResourceFactoryWithoutBlankNodes(new NodeUriGenerator(new CRC32HashGenerator())),
```

Comment:
Could it be that you forgot to use `RdfResourceFactory` from the container which takes into account the `('config')['rdf']['blank_nodes_allowed']` ?

If so, I think that is also wrong on other classes.

Reply:
This is for the test classes, not the actual implementation. Because I updated the expected outputs, this is required.
The downside of course it that currently the unit tests don't test the old versions anymore.
I want to fix that I will have to duplicate all the unit tests for the RDF projections to the old outputs.

Given that I hope this only lives for a week or 2, and given how much code this would double, and needs to be removed later - I prefer this approach.

Reply:
Ignore my comment, because it's not an issue to hardcode the flavour for the unit tests.